examples/workflows/child_workflow_terminated_workflow.rb

```diff
@@ -12,6 +12,9 @@ def execute
       child_workflow_execution.run_id
     )
 
+    # Give time for termination to propagate
+    workflow.sleep(1)
```

Comment:
I'm surprised this is necessary. I'd expect the parent workflow to not replay until the activity has returned and the child has actually been terminated. It may just need another workflow task to be scheduled (which gets caused by `.sleep`) before the history entry for the terminated child finds its way into the parent. I think this is fine for now, but possibly something to look into.

Reply:
That's possible. This was broken in CI and on my machine until I added a `workflow.logger.info` statement here. I switched it to a sleep as a guess that it just needed more time but didn't look into it in depth.